src/sass/claims-status.scss

```diff
@@ -948,3 +948,98 @@ h1:focus {
   }
 }
 
+
+$end-of-docket-width: 4px;
+$docket-bar-width: calc(100% - #{$end-of-docket-width});
+$marker-width: 3rem;
+$marker-text-width: 9rem;
+
+.docket-container {
+  // This is slightly darker than the cards above it
+  background-color: $color-gray-light-alt;
+  padding: 16px;
+  margin: 3em 0px;
+
+  p {
+    margin: 0px;
+  }
+
+  span.appeals-ahead {
```

Comment:
is this selector needed for specificity or would a purely class-based one work?

Reply:
Nah, I was just being specific. :man_shrugging: